content/docs/alerting/configuration.md

```diff
@@ -292,6 +331,9 @@ room_id: <tmpl_string>
 [ message_format:  <string> | default = 'text' ]
 # Background color for message.
 [ color:  <tmpl_string> | default = '{{ if eq .Status "firing" }}red{{ else }}green{{ end }}' ]
+
+# The HTTP client's configuration.
+[ http_config: <http_config> | default = global.http_config ]
```

Comment:
We should have a few words on this up top, as we don't have a struct default anywhere else.

Reply:
Done